mcstatus/tests/test_server.py

```diff
@@ -116,11 +117,13 @@ def test_status(self):
 
         assert self.socket.flush() == bytearray.fromhex("0F002F096C6F63616C686F737463DD01010009010000000001C54246")
         assert self.socket.remaining() == 0, "Data is pending to be read, but should be empty"
-        assert info.raw == {
```

Comment:
I see that `raw` is missing from the object returned by `.status()`. Many users have written code based on the presence of this field; it must be left in.

Reply:
Many libraries (like discord.py) have this field, and it's possible that the protocol specification changes and adds something we won't yet support, so the only way to get it would be through this field, another reason to keep it is that a minecraft server can return something custom, or afterall there are several custom server implementations, and our library just might not be able to parse these custom fields, even though we received it (as the server just sends this JSON string). Without it, this additional context could be lost. Let's keep it in.

Reply:
We shouldn't support something other from official API. Changes like adding a custom field inside a server's answer breaks the API, as many can just do `**answer` and everything will explode.

#319 is very same, as you described. Server implementation changed the API in a non-standard way, so everything exploded. However, this behavior can be expected, default implementation can't return an answer like this.

Reply:
I returned `raw` in 860f7eec1a49776fd4c704c4b9d5bc1424d2660f.